tests/Satooshi/Bundle/CoverallsV1Bundle/Config/ConfiguratorTest.php

```diff
@@ -41,8 +41,8 @@ protected function tearDown()
 
     protected function assertConfiguration(Configuration $config, array $cloverXml, $jsonPath, $excludeNoStatements = false)
     {
-        $this->assertSame($cloverXml, $config->getCloverXmlPaths());
-        $this->assertSame($jsonPath, $config->getJsonPath());
+        $this->assertSamePaths($cloverXml, $config->getCloverXmlPaths());
+        $this->assertSamePath($jsonPath, $config->getJsonPath());
```

Comment:
IMO, having `/` everywhere in code is nicer than need to use `DIRECTORY_SEPARATOR` everywhere, I find later messy.
both `\` and `/` works properly on Windows since ages, we only need to handle that in tests